python_options_tests.erl:68: The created fun has no local return
python_options_tests.erl:68: The created fun has no local return
python_options_tests.erl:69: The call python_options:parse([{'compressed','invalid'},...]) breaks the contract (Options::options()) -> {'ok',#python_options{}} | {'error',Reason::term()}
python_options_tests.erl:93: The created fun has no local return
python_options_tests.erl:93: The created fun has no local return
python_options_tests.erl:94: The call python_options:parse([{'start_timeout','invalid'},...]) breaks the contract (Options::options()) -> {'ok',#python_options{}} | {'error',Reason::term()}
python_options_tests.erl:106: The created fun has no local return
python_options_tests.erl:106: The created fun has no local return
python_options_tests.erl:107: The call python_options:parse([{'call_timeout','invalid'},...]) breaks the contract (Options::options()) -> {'ok',#python_options{}} | {'error',Reason::term()}
python_options_tests.erl:116: The created fun has no local return
python_options_tests.erl:116: The created fun has no local return
python_options_tests.erl:121: The created fun has no local return
python_options_tests.erl:121: The created fun has no local return
python_options_tests.erl:122: The call python_options:parse([{'env','invalid_env'},...]) breaks the contract (Options::options()) -> {'ok',#python_options{}} | {'error',Reason::term()}
python_options_tests.erl:179: The created fun has no local return
python_options_tests.erl:179: The created fun has no local return
python_options_tests.erl:180: The call python_options:parse([{'python','not_string'},...]) breaks the contract (Options::options()) -> {'ok',#python_options{}} | {'error',Reason::term()}
python_options_tests.erl:327: The created fun has no local return
python_options_tests.erl:327: The created fun has no local return
python_options_tests.erl:329: The call python_options:parse([{'python_path','invalid_path'},...]) breaks the contract (Options::options()) -> {'ok',#python_options{}} | {'error',Reason::term()}
python_options_tests.erl:333: The created fun has no local return
python_options_tests.erl:333: The created fun has no local return
python_options_tests.erl:336: The created fun has no local return
python_options_tests.erl:336: The created fun has no local return
python_options_tests.erl:351: The created fun has no local return
python_options_tests.erl:351: The created fun has no local return
python_options_tests.erl:352: The call python_options:parse(['unknown',...]) breaks the contract (Options::options()) -> {'ok',#python_options{}} | {'error',Reason::term()}
python_tests.erl:242: Cons will produce an improper list since its 2nd argument is 1
python_tests.erl:243: Cons will produce an improper list since its 2nd argument is nonempty_improper_list(2 | 3,4)
python_tests.erl:244: Cons will produce an improper list since its 2nd argument is 'tail'
ruby_options_tests.erl:68: The created fun has no local return
ruby_options_tests.erl:68: The created fun has no local return
ruby_options_tests.erl:69: The call ruby_options:parse([{'compressed','invalid'},...]) breaks the contract (Options::options()) -> {'ok',#ruby_options{}} | {'error',Reason::term()}
ruby_options_tests.erl:93: The created fun has no local return
ruby_options_tests.erl:93: The created fun has no local return
ruby_options_tests.erl:94: The call ruby_options:parse([{'start_timeout','invalid'},...]) breaks the contract (Options::options()) -> {'ok',#ruby_options{}} | {'error',Reason::term()}
ruby_options_tests.erl:106: The created fun has no local return
ruby_options_tests.erl:106: The created fun has no local return
ruby_options_tests.erl:107: The call ruby_options:parse([{'call_timeout','invalid'},...]) breaks the contract (Options::options()) -> {'ok',#ruby_options{}} | {'error',Reason::term()}
ruby_options_tests.erl:116: The created fun has no local return
ruby_options_tests.erl:116: The created fun has no local return
ruby_options_tests.erl:121: The created fun has no local return
ruby_options_tests.erl:121: The created fun has no local return
ruby_options_tests.erl:122: The call ruby_options:parse([{'env','invalid_env'},...]) breaks the contract (Options::options()) -> {'ok',#ruby_options{}} | {'error',Reason::term()}
ruby_options_tests.erl:176: The created fun has no local return
ruby_options_tests.erl:176: The created fun has no local return
ruby_options_tests.erl:177: The call ruby_options:parse([{'ruby','not_string'},...]) breaks the contract (Options::options()) -> {'ok',#ruby_options{}} | {'error',Reason::term()}
ruby_options_tests.erl:321: The created fun has no local return
ruby_options_tests.erl:321: The created fun has no local return
ruby_options_tests.erl:323: The call ruby_options:parse([{'ruby_lib','invalid_path'},...]) breaks the contract (Options::options()) -> {'ok',#ruby_options{}} | {'error',Reason::term()}
ruby_options_tests.erl:327: The created fun has no local return
ruby_options_tests.erl:327: The created fun has no local return
ruby_options_tests.erl:330: The created fun has no local return
ruby_options_tests.erl:330: The created fun has no local return
ruby_options_tests.erl:345: The created fun has no local return
ruby_options_tests.erl:345: The created fun has no local return
ruby_options_tests.erl:346: The call ruby_options:parse(['unknown',...]) breaks the contract (Options::options()) -> {'ok',#ruby_options{}} | {'error',Reason::term()}
